tests/ssl-opt.sh

```diff
@@ -7720,6 +7720,8 @@ run_test    "Large server packet TLS 1.2 AEAD shorter tag" \
 
 # Tests for restartable ECC
 
+requires_config_enabled MBEDTLS_ECP_DP_SECP256R1_ENABLED
+requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
```

Comment:
It might be useful to add a 'protocols' domain of tests to depends.py :)

Reply:
Aha. Fortunately in 3.x we only have TLS 1.2 and TLS 1.3. IIRC so far 1.3 is not in the default config, so that gives us a lot of 1.2-only builds for free, then all builds with full config have both protocols, then we have `component_test_tls13_only` in `all.sh` so I think we're good. (We also have 1.2-only builds from `test-ref-configs.pl`, for when we finally enable 1.3 by default.)

But in 2.28 IIRC we have 4 protocols (SSL 3.0 to TLS 1.2), and I don't think we have systematic testing with only one of them. We have some 1.2-only and 1.1-only builds via `test-ref-config.pl`, but I don't think we have 1.0-only or SSL3-only build. Though hopefully at this point nobody wants to use such builds :)